src/components/DetailsList/DetailsList.hbs

```diff
@@ -0,0 +1,111 @@
+<!-- Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information. -->
```

Comment:
Turns out it's not related to the Breadcrumb, but is a necessary file to get the build working.